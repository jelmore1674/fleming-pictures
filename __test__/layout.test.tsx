import React from 'react';
import { render, screen, fireEvent, act } from '@testing-library/react';
import Layout from '../layout/layout';
import Home from '../pages/index';

describe('<Layout />', () => {
	it('should match snapshot', () => {
		const layout = render(
			<Layout>
				<Home />
			</Layout>
		);
		expect(layout).toMatchSnapshot();
	});
});
