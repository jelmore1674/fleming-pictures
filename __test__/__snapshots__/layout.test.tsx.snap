// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="nav"
        >
          <div
            class="nav-item about-nav-item"
            data-testid="/about"
          >
            <a
              href="/about"
            >
              About
            </a>
          </div>
          <div
            class="logo"
            data-testid="logo"
          >
            <div
              href="/"
            >
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                    style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                  />
                </div>
                <img
                  alt="logo"
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </div>
            </div>
          </div>
          <div
            class="nav-item connect-nav-item"
            data-testid="/connect"
          >
            <a
              href="/connect"
            >
              Connect
            </a>
          </div>
        </div>
      </nav>
      <main>
        <div>
          <div
            class="featured-slideshow"
            data-testid="featured-section"
          >
            <div
              class="featured-section"
              data-testid="feature-img-0"
              style="transform: translate3d(0%, 0, 0);"
            >
              <div
                class="featured-slider"
                data-testid="img-container0"
              >
                <div
                  class="featured-image"
                  style="background-image: url(/assets/images/chronus1920x1080.png);"
                >
                  <button
                    class="modal-button"
                    data-testid="btn-0"
                    id="btn-0"
                    role="button"
                  >
                    More Info
                  </button>
                </div>
              </div>
              <div
                class="featured-slider"
                data-testid="img-container1"
              >
                <div
                  class="featured-image"
                  style="background-image: url(/assets/images/featured2.png);"
                >
                  <button
                    class="modal-button"
                    data-testid="btn-1"
                    id="btn-1"
                    role="button"
                  >
                    More Info
                  </button>
                </div>
              </div>
            </div>
            <div
              class="slideshowDots"
            >
              <div
                class="slideshowDot active"
                data-testid="slider-status-0"
              />
              <div
                class="slideshowDot"
                data-testid="slider-status-1"
              />
            </div>
            <div
              class="icon icon-left"
              data-testid="cursor-left"
            >
              <i
                class="fas fa-chevron-left"
              />
            </div>
            <div
              class="icon icon-right"
              data-testid="cursor-right"
            >
              <i
                class="fas fa-chevron-right"
              />
            </div>
          </div>
          <div
            class="poster-grid"
          >
            <h3
              class="heading"
            >
              Other Films
            </h3>
            <div
              class="grid-container"
            >
              <div
                class="poster-item"
              >
                <div
                  class="poster-image"
                >
                  <div
                    style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
                  >
                    <div
                      style="box-sizing: border-box; display: block; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIzIiBoZWlnaHQ9IjMzMyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                        style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                      />
                    </div>
                    <img
                      alt="poster"
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <div
                  class="film-title"
                >
                  The Chronus
                </div>
              </div>
              <div
                class="poster-item"
              >
                <div
                  class="poster-image"
                >
                  <div
                    style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
                  >
                    <div
                      style="box-sizing: border-box; display: block; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIzIiBoZWlnaHQ9IjMzMyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                        style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                      />
                    </div>
                    <img
                      alt="poster"
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <div
                  class="film-title"
                >
                  Another Film
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      <footer
        class="footer"
      >
        <div
          data-testid="copyright"
        >
          Copyright 2021
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="nav"
      >
        <div
          class="nav-item about-nav-item"
          data-testid="/about"
        >
          <a
            href="/about"
          >
            About
          </a>
        </div>
        <div
          class="logo"
          data-testid="logo"
        >
          <div
            href="/"
          >
            <div
              style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
            >
              <div
                style="box-sizing: border-box; display: block; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                  style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                />
              </div>
              <img
                alt="logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </div>
          </div>
        </div>
        <div
          class="nav-item connect-nav-item"
          data-testid="/connect"
        >
          <a
            href="/connect"
          >
            Connect
          </a>
        </div>
      </div>
    </nav>
    <main>
      <div>
        <div
          class="featured-slideshow"
          data-testid="featured-section"
        >
          <div
            class="featured-section"
            data-testid="feature-img-0"
            style="transform: translate3d(0%, 0, 0);"
          >
            <div
              class="featured-slider"
              data-testid="img-container0"
            >
              <div
                class="featured-image"
                style="background-image: url(/assets/images/chronus1920x1080.png);"
              >
                <button
                  class="modal-button"
                  data-testid="btn-0"
                  id="btn-0"
                  role="button"
                >
                  More Info
                </button>
              </div>
            </div>
            <div
              class="featured-slider"
              data-testid="img-container1"
            >
              <div
                class="featured-image"
                style="background-image: url(/assets/images/featured2.png);"
              >
                <button
                  class="modal-button"
                  data-testid="btn-1"
                  id="btn-1"
                  role="button"
                >
                  More Info
                </button>
              </div>
            </div>
          </div>
          <div
            class="slideshowDots"
          >
            <div
              class="slideshowDot active"
              data-testid="slider-status-0"
            />
            <div
              class="slideshowDot"
              data-testid="slider-status-1"
            />
          </div>
          <div
            class="icon icon-left"
            data-testid="cursor-left"
          >
            <i
              class="fas fa-chevron-left"
            />
          </div>
          <div
            class="icon icon-right"
            data-testid="cursor-right"
          >
            <i
              class="fas fa-chevron-right"
            />
          </div>
        </div>
        <div
          class="poster-grid"
        >
          <h3
            class="heading"
          >
            Other Films
          </h3>
          <div
            class="grid-container"
          >
            <div
              class="poster-item"
            >
              <div
                class="poster-image"
              >
                <div
                  style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
                >
                  <div
                    style="box-sizing: border-box; display: block; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIzIiBoZWlnaHQ9IjMzMyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                      style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                    />
                  </div>
                  <img
                    alt="poster"
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </div>
              </div>
              <div
                class="film-title"
              >
                The Chronus
              </div>
            </div>
            <div
              class="poster-item"
            >
              <div
                class="poster-image"
              >
                <div
                  style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
                >
                  <div
                    style="box-sizing: border-box; display: block; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIzIiBoZWlnaHQ9IjMzMyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                      style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                    />
                  </div>
                  <img
                    alt="poster"
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </div>
              </div>
              <div
                class="film-title"
              >
                Another Film
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer"
    >
      <div
        data-testid="copyright"
      >
        Copyright 2021
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
